src/server.ts

```diff
@@ -59,7 +59,11 @@ export const start = (options: StartOptions): Promise<any> => {
 
   server.express.use(authorizationMiddleware(settings.accessToken));
   // tslint:disable-next-line:no-console
-  return server.start({ port: settings.port, formatError: (errors: any) => {
+  return server.start({
+      // fix webpack doesn't bundle subscriptions
+      subscriptions: false,
```

Comment:
do we even have subscriptions?

Reply:
Nope... still a sad fix but that's all I have ATM lol